docs/CLIReadme.md

```diff
@@ -11,144 +11,154 @@ For Zowe CLI support, visit https://zowe.org
-		* [data-set](#command-data-set)
-		* [uss-files](#command-uss-files)
-	* [upload | ul](#module-upload)
-		* [file-to-data-set](#command-file-to-data-set)
-		* [stdin-to-data-set](#command-stdin-to-data-set)
-		* [dir-to-pds](#command-dir-to-pds)
-		* [file-to-uss](#command-file-to-uss)
-		* [dir-to-uss](#command-dir-to-uss)
-* [zos-jobs | jobs](#module-zos-jobs)
-	* [submit | sub](#module-submit)
-		* [data-set](#command-data-set)
-		* [local-file](#command-local-file)
-	* [download | dl](#module-download)
-		* [output](#command-output)
-	* [view | vw](#module-view)
-		* [job-status-by-jobid](#command-job-status-by-jobid)
-		* [spool-file-by-id](#command-spool-file-by-id)
-	* [list | ls](#module-list)
-		* [spool-files-by-jobid](#command-spool-files-by-jobid)
-		* [jobs](#command-jobs)
-	* [delete | del](#module-delete)
-		* [job](#command-job)
-	* [cancel | can](#module-cancel)
-		* [job](#command-job)
-* [zos-tso | tso](#module-zos-tso)
-	* [send](#module-send)
-		* [address-space](#command-address-space)
-	* [start | st](#module-start)
-		* [address-space](#command-address-space)
-	* [ping](#module-ping)
-		* [address-space](#command-address-space)
-	* [stop | sp](#module-stop)
-		* [address-space](#command-address-space)
-	* [issue](#module-issue)
-		* [command](#command-command)
-* [zos-uss | uss](#module-zos-uss)
-	* [issue | iss](#module-issue)
-		* [ssh](#command-ssh)
-* [zos-workflows | wf](#module-zos-workflows)
-	* [create | cre](#module-create)
-		* [workflow-from-data-set](#command-workflow-from-data-set)
-		* [workflow-from-uss-file](#command-workflow-from-uss-file)
-		* [workflow-from-local-file](#command-workflow-from-local-file)
-	* [start | sta](#module-start)
-		* [workflow-full](#command-workflow-full)
-		* [workflow-step](#command-workflow-step)
-	* [list | ls](#module-list)
-		* [active-workflows](#command-active-workflows)
-		* [active-workflow-details](#command-active-workflow-details)
-		* [definition-file-details](#command-definition-file-details)
-		* [archived-workflows](#command-archived-workflows)
-	* [archive](#module-archive)
-		* [active-workflow](#command-active-workflow)
-	* [delete | del](#module-delete)
-		* [active-workflow](#command-active-workflow)
-		* [archived-workflow](#command-archived-workflow)
-* [zosmf](#module-zosmf)
-	* [check](#module-check)
-		* [status](#command-status)
-	* [list](#module-list)
-		* [systems](#command-systems)
-
-
-# config<a name="module-config"></a>
+* [config](#config)
+	* [set](#config-set)
+	* [reset](#config-reset)
+	* [list](#config-list)
+	* [get](#config-get)
+* [plugins](#plugins)
+	* [install](#plugins-install)
+	* [list](#plugins-list)
+	* [uninstall](#plugins-uninstall)
+	* [update](#plugins-update)
+	* [validate](#plugins-validate)
+* [profiles](#profiles)
+	* [create | cre](# profiles-create)
+		* [zosmf-profile](# profiles-create-zosmf-profile)
+		* [tso-profile](# profiles-create-tso-profile)
+		* [ssh-profile](# profiles-create-ssh-profile)
```

Comment:
I believe the anchors won't work because of the space
![image](https://user-images.githubusercontent.com/37381190/70453204-60c1d400-1a76-11ea-8a4a-03dad722e8bd.png)